<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
	<class>MainWindow</class>
	<widget class="QMainWindow" name="MainWindow">
		<property name="geometry">
			<rect>
				<x>0</x>
				<y>0</y>
				<width>845</width>
				<height>850</height>
			</rect>
		</property>
		<property name="windowTitle">
			<string>Process Scheduling Simulator</string>
		</property>
		<widget class="QWidget" name="centralwidget">
			<widget class="QLabel" name="label">
				<property name="geometry">
					<rect>
						<x>20</x>
						<y>20</y>
						<width>111</width>
						<height>16</height>
					</rect>
				</property>
				<property name="font">
					<font>
						<family>맑은 고딕</family>
						<weight>50</weight>
						<bold>false</bold>
					</font>
				</property>
				<property name="text">
					<string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt; font-weight:600;&quot;&gt;Algorithm List&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
				</property>
			</widget>
			<widget class="QLabel" name="label_3">
				<property name="geometry">
					<rect>
						<x>160</x>
						<y>20</y>
						<width>111</width>
						<height>21</height>
					</rect>
				</property>
				<property name="font">
					<font>
						<family>맑은 고딕</family>
						<weight>50</weight>
						<bold>false</bold>
					</font>
				</property>
				<property name="text">
					<string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt; font-weight:600;&quot;&gt;Arrival time&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
				</property>
			</widget>
			<widget class="QLabel" name="label_4">
				<property name="geometry">
					<rect>
						<x>260</x>
						<y>20</y>
						<width>111</width>
						<height>21</height>
					</rect>
				</property>
				<property name="font">
					<font>
						<family>맑은 고딕</family>
						<weight>50</weight>
						<bold>false</bold>
					</font>
				</property>
				<property name="text">
					<string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt; font-weight:600;&quot;&gt;Burst time&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
				</property>
			</widget>
			<widget class="QComboBox" name="comboBox_Alist">
				<property name="geometry">
					<rect>
						<x>20</x>
						<y>40</y>
						<width>121</width>
						<height>22</height>
					</rect>
				</property>
				<item>
					<property name="text">
						<string>선택</string>
					</property>
				</item>
				<item>
					<property name="text">
						<string>FCFS</string>
					</property>
				</item>
				<item>
					<property name="text">
						<string>RR</string>
					</property>
				</item>
				<item>
					<property name="text">
						<string>SPN</string>
					</property>
				</item>
				<item>
					<property name="text">
						<string>SRTN</string>
					</property>
				</item>
				<item>
					<property name="text">
						<string>HRRN</string>
					</property>
				</item>
				<item>
					<property name="text">
						<string>RPN</string>
					</property>
				</item>
			</widget>
			<widget class="QLineEdit" name="lineEdit_AT">
				<property name="geometry">
					<rect>
						<x>160</x>
						<y>40</y>
						<width>81</width>
						<height>21</height>
					</rect>
				</property>
			</widget>
			<widget class="QLineEdit" name="lineEdit_BT">
				<property name="geometry">
					<rect>
						<x>260</x>
						<y>40</y>
						<width>81</width>
						<height>21</height>
					</rect>
				</property>
			</widget>
			<widget class="QPushButton" name="pushButton_add_process">
				<property name="geometry">
					<rect>
						<x>580</x>
						<y>30</y>
						<width>111</width>
						<height>41</height>
					</rect>
				</property>
				<property name="text">
					<string>Process Add</string>
				</property>
			</widget>
			<widget class="QPushButton" name="pushButton_reset">
				<property name="geometry">
					<rect>
						<x>710</x>
						<y>30</y>
						<width>111</width>
						<height>41</height>
					</rect>
				</property>
				<property name="text">
					<string>Reset</string>
				</property>
			</widget>
			<widget class="QLabel" name="label_5">
				<property name="geometry">
					<rect>
						<x>20</x>
						<y>70</y>
						<width>111</width>
						<height>16</height>
					</rect>
				</property>
				<property name="font">
					<font>
						<family>맑은 고딕</family>
						<weight>50</weight>
						<bold>false</bold>
					</font>
				</property>
				<property name="text">
					<string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;&amp;lt;Time table&amp;gt;&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
				</property>
			</widget>
			<widget class="QLineEdit" name="lineEdit_Type">
				<property name="geometry">
					<rect>
						<x>580</x>
						<y>130</y>
						<width>111</width>
						<height>31</height>
					</rect>
				</property>
			</widget>
			<widget class="QLabel" name="label_9">
				<property name="geometry">
					<rect>
						<x>360</x>
						<y>-10</y>
						<width>111</width>
						<height>81</height>
					</rect>
				</property>
				<property name="font">
					<font>
						<family>맑은 고딕</family>
						<weight>50</weight>
						<bold>false</bold>
					</font>
				</property>
				<property name="text">
					<string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt; font-weight:600;&quot;&gt;Quantum&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
				</property>
			</widget>
			<widget class="QLineEdit" name="lineEdit_quantum">
				<property name="geometry">
					<rect>
						<x>360</x>
						<y>40</y>
						<width>81</width>
						<height>21</height>
					</rect>
				</property>
			</widget>
			<widget class="QTableWidget" name="tableWidget_Timetable">
				<property name="geometry">
					<rect>
						<x>20</x>
						<y>100</y>
						<width>541</width>
						<height>141</height>
					</rect>
				</property>
				<property name="font">
					<font>
						<pointsize>8</pointsize>
					</font>
				</property>
				<attribute name="horizontalHeaderVisible">
					<bool>true</bool>
				</attribute>
				<attribute name="horizontalHeaderCascadingSectionResizes">
					<bool>false</bool>
				</attribute>
				<attribute name="horizontalHeaderDefaultSectionSize">
					<number>170</number>
				</attribute>
				<attribute name="horizontalHeaderShowSortIndicator" stdset="0">
					<bool>false</bool>
				</attribute>
				<attribute name="horizontalHeaderStretchLastSection">
					<bool>true</bool>
				</attribute>
				<column>
					<property name="text">
						<string>Process Number</string>
					</property>
				</column>
				<column>
					<property name="text">
						<string>Arrival Time(AT)</string>
					</property>
				</column>
				<column>
					<property name="text">
						<string>Burst Time(BT)</string>
					</property>
				</column>
			</widget>
			<widget class="QLabel" name="label_11">
				<property name="geometry">
					<rect>
						<x>20</x>
						<y>240</y>
						<width>121</width>
						<height>21</height>
					</rect>
				</property>
				<property name="font">
					<font>
						<family>맑은 고딕</family>
						<weight>50</weight>
						<bold>false</bold>
					</font>
				</property>
				<property name="text">
					<string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;&amp;lt;Ready Queue&amp;gt;&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
				</property>
			</widget>
			<widget class="QLabel" name="label_12">
				<property name="geometry">
					<rect>
						<x>20</x>
						<y>370</y>
						<width>111</width>
						<height>16</height>
					</rect>
				</property>
				<property name="font">
					<font>
						<family>맑은 고딕</family>
						<weight>50</weight>
						<bold>false</bold>
					</font>
				</property>
				<property name="text">
					<string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;&amp;lt;Gantt Chart&amp;gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
				</property>
			</widget>
			<widget class="QLabel" name="label_time">
				<property name="geometry">
					<rect>
						<x>630</x>
						<y>350</y>
						<width>141</width>
						<height>51</height>
					</rect>
				</property>
				<property name="font">
					<font>
						<family>맑은 고딕</family>
						<weight>50</weight>
						<bold>false</bold>
					</font>
				</property>
				<property name="text">
					<string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Real time = 0 sec&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
				</property>
			</widget>
			<widget class="QTableWidget" name="tableWidget_gantt">
				<property name="geometry">
					<rect>
						<x>20</x>
						<y>390</y>
						<width>801</width>
						<height>231</height>
					</rect>
				</property>
				<attribute name="verticalHeaderDefaultSectionSize">
					<number>85</number>
				</attribute>
				<attribute name="verticalHeaderStretchLastSection">
					<bool>false</bool>
				</attribute>
			</widget>
			<widget class="QLabel" name="label_14">
				<property name="geometry">
					<rect>
						<x>20</x>
						<y>630</y>
						<width>111</width>
						<height>21</height>
					</rect>
				</property>
				<property name="font">
					<font>
						<family>맑은 고딕</family>
						<weight>50</weight>
						<bold>false</bold>
					</font>
				</property>
				<property name="text">
					<string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;&amp;lt;Result table&amp;gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
				</property>
			</widget>
			<widget class="QTableWidget" name="tableWidget_result">
				<property name="geometry">
					<rect>
						<x>20</x>
						<y>651</y>
						<width>801</width>
						<height>141</height>
					</rect>
				</property>
				<property name="font">
					<font>
						<pointsize>8</pointsize>
					</font>
				</property>
				<attribute name="horizontalHeaderDefaultSectionSize">
					<number>130</number>
				</attribute>
				<attribute name="horizontalHeaderStretchLastSection">
					<bool>true</bool>
				</attribute>
				<column>
					<property name="text">
						<string>Process ID</string>
					</property>
				</column>
				<column>
					<property name="text">
						<string>AT</string>
					</property>
				</column>
				<column>
					<property name="text">
						<string>BT</string>
					</property>
				</column>
				<column>
					<property name="text">
						<string>WT</string>
					</property>
				</column>
				<column>
					<property name="text">
						<string>TT</string>
					</property>
				</column>
				<column>
					<property name="text">
						<string>NTT</string>
					</property>
				</column>
			</widget>
			<widget class="QTableWidget" name="tableWidget_queue">
				<property name="geometry">
					<rect>
						<x>20</x>
						<y>260</y>
						<width>801</width>
						<height>91</height>
					</rect>
				</property>
				<attribute name="horizontalHeaderStretchLastSection">
					<bool>false</bool>
				</attribute>
				<attribute name="verticalHeaderStretchLastSection">
					<bool>true</bool>
				</attribute>
			</widget>
			<widget class="QPushButton" name="pushButton_next">
				<property name="geometry">
					<rect>
						<x>760</x>
						<y>360</y>
						<width>51</width>
						<height>28</height>
					</rect>
				</property>
				<property name="text">
					<string>→</string>
				</property>
			</widget>
			<widget class="QPushButton" name="pushButton_prev">
				<property name="geometry">
					<rect>
						<x>570</x>
						<y>360</y>
						<width>51</width>
						<height>28</height>
					</rect>
				</property>
				<property name="text">
					<string>←</string>
				</property>
			</widget>
			<widget class="QLabel" name="label_8">
				<property name="geometry">
					<rect>
						<x>580</x>
						<y>100</y>
						<width>121</width>
						<height>31</height>
					</rect>
				</property>
				<property name="font">
					<font>
						<family>맑은 고딕</family>
						<weight>50</weight>
						<bold>false</bold>
					</font>
				</property>
				<property name="text">
					<string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt; font-weight:600;&quot;&gt;Processor Type&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
				</property>
			</widget>
			<widget class="QPushButton" name="pushButton_add_processor">
				<property name="geometry">
					<rect>
						<x>710</x>
						<y>100</y>
						<width>111</width>
						<height>61</height>
					</rect>
				</property>
				<property name="text">
					<string>CPU Add</string>
				</property>
			</widget>
			<widget class="QPushButton" name="pushButton_start">
				<property name="geometry">
					<rect>
						<x>580</x>
						<y>180</y>
						<width>241</width>
						<height>61</height>
					</rect>
				</property>
				<property name="text">
					<string>Start</string>
				</property>
			</widget>
			<widget class="QLabel" name="label_Priority">
				<property name="geometry">
					<rect>
						<x>460</x>
						<y>20</y>
						<width>111</width>
						<height>21</height>
					</rect>
				</property>
				<property name="font">
					<font>
						<family>맑은 고딕</family>
						<weight>50</weight>
						<bold>false</bold>
					</font>
				</property>
				<property name="text">
					<string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt; font-weight:600;&quot;&gt;Priority&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
				</property>
			</widget>
			<widget class="QLabel" name="label_Watt">
				<property name="geometry">
					<rect>
						<x>700</x>
						<y>630</y>
						<width>171</width>
						<height>16</height>
					</rect>
				</property>
				<property name="text">
					<string>소비전력 : 0 W </string>
				</property>
			</widget>
			<widget class="QComboBox" name="comboBox_Priority">
				<property name="geometry">
					<rect>
						<x>460</x>
						<y>40</y>
						<width>101</width>
						<height>22</height>
					</rect>
				</property>
				<item>
					<property name="text">
						<string>선택</string>
					</property>
				</item>
				<item>
					<property name="text">
						<string>실시간</string>
					</property>
				</item>
				<item>
					<property name="text">
						<string>높음</string>
					</property>
				</item>
				<item>
					<property name="text">
						<string>높은우선순위</string>
					</property>
				</item>
				<item>
					<property name="text">
						<string>보통</string>
					</property>
				</item>
				<item>
					<property name="text">
						<string>낮은우선순위</string>
					</property>
				</item>
				<item>
					<property name="text">
						<string>낮음</string>
					</property>
				</item>
			</widget>
		</widget>
		<widget class="QMenuBar" name="menubar">
			<property name="geometry">
				<rect>
					<x>0</x>
					<y>0</y>
					<width>845</width>
					<height>26</height>
				</rect>
			</property>
		</widget>
		<widget class="QStatusBar" name="statusbar"/>
	</widget>
	<resources/>
	<connections/>
	<slots>
		<slot>start()</slot>
		<slot>add()</slot>
		<slot>reset()</slot>
	</slots>
</ui>